src/components/frontend-engine/types.ts

```diff
@@ -142,6 +142,8 @@ export interface IFrontendEngineRef
 	reset: UseFormReset<TFrontendEngineValues>;
 	/** allows setting of custom errors */
 	setErrors: (errors: TErrorPayload) => void;
+	/** clear form errors */
+	clearErrors: (error?: string | string[] | undefined) => void;
```

Comment:
follow react-hook-form's typings

```suggestion
	clearErrors: UseFormClearErrors<TFrontendEngineValues>;
```